### PR TITLE
[#1087] Update badges in README

### LINKED ISSUE #1087: README: update netlify badge
Starting April 14, 2020, sites without a custom domain are being moved from your-site.netlify.com to your-site.netlify.app. As the badge icon on our readme is a customed icon, it would not work after 14 April. See if netlify have its own badge icon now that we can implement or update the current badge on 14 April.

### PR DESCRIPTION
**Pull request by jamessspanggg**
Fixes #1087

```
RepoSense's Netlify badge icon is a custom icon. We are missing 
acknowledgement to Surge as well.

Let's update the badge to Netlify's official badge that reflects our
Netlify deployment status. In the meantime, let's also acknowledge
Surge in helping with our web previews.
```

### CODE REVIEW

**Comment by jamessspanggg** [comment]
@eugenepeh since u are disabling netlify, should we still put the netlify badge?

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 [![Build Status](https://travis-ci.org/reposense/RepoSense.svg?branch=master)](https://travis-ci.org/reposense/RepoSense)
 [![Build status](https://ci.appveyor.com/api/projects/status/gsbkj5qby3pjd6nw/branch/master?svg=true)](https://ci.appveyor.com/project/eugenepeh/reposense/branch/master)
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/08a3527378464ed4a5ad62e27f590d6a)](https://www.codacy.com/app/reposense/RepoSense?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=reposense/RepoSense&amp;utm_campaign=Badge_Grade)
-[![deploy on_netlify](https://img.shields.io/badge/deploy-on_netlify-blue.svg)](https://reposense.netlify.com/)
+[![Netlify Status](https://api.netlify.com/api/v1/badges/260983b3-589e-4619-a2e8-0bfb7a2b4422/deploy-status)](https://app.netlify.com/sites/reposense/deploys)
```
Comment (eugenepeh) [inline README.md]: its style is a bit different from others, let's put it at the end instead?

Comment (jamessspanggg) [reply README.md]: sure

Comment (jamessspanggg) [reply README.md]: done.

**Comment by jamessspanggg** [comment]
@eugenepeh should we add ... powered by Netlify and Surge? As well as the surge logo?

Screen Shot 2020-03-29 at 1 46 14 PM

**Comment by eugenepeh** [comment]
>@eugenepeh should we add ... powered by Netlify and Surge? As well as the surge logo?

The powered by Netlify is a requirement given by Netlify to be qualified for the open source plan. We have to retain it to enjoy the service. As for Surge, we can include it as a gesture of good faith.

**Comment by fzdy1914** [comment]
Actually, I don't there is a need to change to the badge to the official one. We do not care so much about the status of the deployment anyway. We only what an entry point to our website.

**Comment by jamessspanggg** [comment]
@fzdy1914 

> Starting April 14, 2020, sites without a custom domain are being moved from your-site.netlify.com to your-site.netlify.app. As the badge icon on our readme is a customed icon, it would not work after 14 April. See if netlify have its own badge icon now that we can implement or update the current badge on 14 April.

Taken from the original issue #1087

**Comment by fzdy1914** [comment]
OK, then its fine

**Comment by jamessspanggg** [comment]
@fzdy1914 then may I ask you to merge this change? don't think need to go through stage 0 and stage 1 reviewers since it's a simple change.

**Pull request merged**